doc/source/whatsnew/v0.24.0.rst

```diff
@@ -795,6 +795,9 @@ Now, the return type is consistently a :class:`DataFrame`.
    and a :class:`DataFrame` with sparse values. The memory usage will
    be the same as in the previous version of pandas.
 
+   Be sure to perform a replace of literal strings by passing the
```

Comment:
Can you move this to the section on breaking changes and show a before / after of the behavior?